README.md

```diff
@@ -53,7 +54,11 @@ Using the `KEYCHAIN_PW` env var:
   plugins:
     - improbable-eng/mac-codesign#v0.1.2:
         input_artifacts:
```

Comment:
Update this to now be singular instead of plural? (Also in the rest of the examples below)

```suggestion
        input_artifact:
```